src/clarity-angular/tabs/_tabs.clarity.scss

```diff
@@ -5,6 +5,12 @@
 @import "../utils/mixins.clarity";
 
 @include exports('tabs.clarity') {
+  @media screen {
+    section[aria-hidden="true"] {
+      display: none;
+    }
+  }
+
   [data-hidden="true"] {
```

Comment:
Sounds good. I will leave it in for this PR. I did check and only the wizard was using it, so I think it'll get removed cleanly in 0.9.